thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/anomaly/monitor/MonitorTaskRunner.java

```diff
@@ -64,9 +64,19 @@
 
   private void executeMonitorUpdate(MonitorTaskInfo monitorTaskInfo) {
     LOG.info("Execute monitor update {}", monitorTaskInfo);
+    int jobRetentionDays = monitorTaskInfo.getDefaultRetentionDays();
     try {
+      // Mark expired task as TIMEOUT
+      List<TaskDTO> timeoutTasks = DAO_REGISTRY.getTaskDAO().findTimeoutTasksWithinDays(jobRetentionDays, MAX_TASK_TIME);
+      if (!timeoutTasks.isEmpty()) {
+        for (TaskDTO task : timeoutTasks) {
+          DAO_REGISTRY.getTaskDAO().updateStatusAndTaskEndTime(task.getId(), TaskStatus.RUNNING, TaskStatus.TIMEOUT,
+              System.currentTimeMillis(), "TIMEOUT status updated by MonitorTask");
+        }
+        LOG.info("TIMEOUT tasks {}", timeoutTasks);
```

Comment:
Let's add more info in the log for debugging and this should be a warning.
This should be really rare.

Reply:
Changed to `warn` instead of `info`.